styles/styles.css

```diff
@@ -29,18 +29,41 @@ header p {
   margin: 0;
 }
 
+
+/* Match section content to height of viewport */
+
+
+@media (min-height: 750px) {
+  .storyline {
+    margin: 100px inherit;
+  }
+}
+/*
+@media (max-height: 900px) {
+  .storyline {
+    margin: 700px 0;
+  }
+}
+
+@media (max-height: 1200px) {
+  .storyline  {
+    margin: 1000px 0;
+  }
+} */
+
+/* Format sections of the storyline */
 .storyline {
   display: flex;
 }
 
-.story-scene {
+.story-divider {
+  margin-top: 5%;
   margin-bottom: 20px;
```

Comment:
https://getbootstrap.com/docs/4.1/utilities/spacing/

The benefit of using spacing is that everything would be pixel aligned. If you determine a space to be 4px, other things would come in multiples for 4px.

Reply:
Haha sorry to make you send me the same link twice. I read over this when you sent it the first time, but apparently did not understand it. I thought the bootstrap notation was only a short-hand version of what I used. I'll take another swing at their notation to see how it changes things.